model/flow/constants.go

```diff
@@ -19,19 +19,28 @@ const DefaultTransactionExpiryBuffer = 30
 // DefaultMaxGasLimit is the default maximum value for the transaction gas limit.
 const DefaultMaxGasLimit = 9999
 
-// DefaultMaxTxSizeLimit is the default maximum transaction byte size. (1.5MB)
-const DefaultMaxTxSizeLimit = 1500000
+// DefaultMaxTxSizeLimit is the default maximum transaction byte size. (~1.5MB)
+const DefaultMaxTxSizeLimit = 1_500_000
+
+// DefaultMaxCollectionByteSize is the default maximum value for a collection byte size.
+const DefaultMaxCollectionByteSize = 3_000_000 // ~3MB. This is should always be higher than the limit on single tx size.
+
+// DefaultMaxCollectionTotalGas is the default maximum value for total gas allowed to be included in a collection.
+const DefaultMaxCollectionTotalGas = 10_000_000 // 10M
+
+// DefaultMaxCollectionSize is the default maximum number of transactions allowed inside a collection.
+const DefaultMaxCollectionSize = 100
```

Comment:
I moved these default values here, since soon we have to check them at execution nodes as well.